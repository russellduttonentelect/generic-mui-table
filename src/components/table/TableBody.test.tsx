import { render } from '@testing-library/react';
import { TableBody } from '.';
// import { Product } from 'src/types';
import { Table } from '@mui/material';

describe('Table Body tests', () => {
  const rows = [
    {
      id: '1',
      lotId: '#1',
      productName: 'First Product'
    }
  ];
  test('should render the table body correctly with sensible parameters', () => {
    const { baseElement } = render(
      <Table>
        <TableBody rows={rows} headers={[]} />
      </Table>
    );

    expect(baseElement).toMatchSnapshot();
  });
});
