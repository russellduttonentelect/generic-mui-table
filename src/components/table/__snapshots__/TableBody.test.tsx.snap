// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table Body tests should render the table body correctly with sensible parameters 1`] = `
<body>
  <div>
    <table
      class="MuiTable-root css-rqglhn-MuiTable-root"
    >
      <tbody
        class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
      >
        <tr
          class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
          data-testid="row-0"
          tabindex="-1"
        />
      </tbody>
    </table>
  </div>
</body>
`;
